main/tests/server/src/com/google/refine/model/recon/StandardReconConfigTests.java

```diff
@@ -61,8 +62,8 @@ public class StandardReconConfigTests extends RefineTest {
     
     @BeforeMethod
     public void registerOperation() {
-        OperationRegistry.registerOperation(getCoreModule().getName(), "recon", ReconOperation.class);
-        ReconConfig.registerReconConfig(getCoreModule().getName(), "standard-service", StandardReconConfig.class);
+        OperationRegistry.registerOperation("core", "recon", ReconOperation.class);
+        ReconConfig.registerReconConfig("core", "standard-service", StandardReconConfig.class);
```

Comment:
I am keen to migrate to something else, but I think it's a bigger change indeed (for instance because it requires changes to Butterfly itself): https://forum.openrefine.org/t/improving-the-ux-of-extension-install-and-butterfly/52